docs/reference/editor-settings.md

```diff
@@ -131,3 +131,34 @@ The extension uses the first executable that exists. This setting takes preceden
   "ty.path": ["/home/user/.local/bin/ty"]
 }
 ```
+
+## Neovim
+
+Assuming that the command `ty` is available in your session, the easiest way to start the server is by adding this to your configuration (e.g. `init.lua`):
+
+```lua
+vim.api.nvim_create_autocmd({ 'BufNewFile', 'BufRead' }, {
+    pattern = '*.py',
+    callback = function()
+        vim.lsp.start({
+            name = 'ty',
+            cmd = { 'ty', 'server' },
+        })
+    end,
+})
+
+```
+
+Alternatively, you can add a custom language server like this (assuming you are using `lspconfig`):
+
+```lua
+local configs = require('lspconfig.configs')
+configs.ty = {
+    default_config = {
+        cmd = { 'ty', 'server' },
+        filetypes = { 'python' },
+        settings = {},
+    },
+}
```

Comment:
I don't think this is required given that the plugin includes the ty config: https://github.com/neovim/nvim-lspconfig/blob/ac1dfbe3b60e5e23a2cff90e3bd6a3bc88031a57/lsp/ty.lua#L4

Reply:
You are right. Now it is correct and follows instructions as in Ruff.